Fabric.Terminology.Domain/Fabric.Terminology.Domain.csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="CallMeMaybe" Version="0.6.6" />
+    <PackageReference Include="CallMeMaybe" Version="0.6.6.42-alpha" />
```

Comment:
It looks like you switched back to the MyGet-hosted prerelease version here.

Reply:
Thanks - I'll double check.